code/modules/materials/material_sheets.dm

```diff
@@ -390,6 +394,13 @@
 	stack_merge_type = /obj/item/stack/material/lump
 	crafting_stack_type = /obj/item/stack/material/lump
 
+/obj/item/stack/material/lump/clay
```

Comment:
Are we sure this works properly? Some thing like SSmaterials might create `/obj/item/stack/material/lump` with the clay material, which uses `"lump"` instead of `"lump_large"`.